glances/plugins/glances_sensors.py

```diff
@@ -127,14 +127,20 @@ def update(self):
 
             pass
 
-        # Set the alias for each stat
+        # Update the stats
+        self.stats = []
```

Comment:
I had no idea what would be the best, and good, way to implement the hide option. I have supposed there is no sync issue doing this way, using no temporary variable.